documentation/commands/git-extract.md

```diff
@@ -6,7 +6,7 @@ git-extract - copy selected commits from the current branch into their own branc
 #### SYNOPSIS
 
 ```
-git extract <branchname> [<commit>...]
+git extract <branch_name> [<commit>...]
```

Comment:
This isn't source code. How do you feel about using `branch name` here (with a space)?